spec/factories/audio_recording_factory.rb

```diff
@@ -105,7 +105,19 @@
       end
     end
 
+    trait :with_date_range do
+      transient do
+        start_date { '2025-01-01 T00:00:00Z' }
```

Comment:
if you just make this a date value you don't have to parse it on each sequence generation step